bin.src/showPsfs.py

```diff
@@ -356,26 +362,26 @@ def frameInfoFrom(filepath):
                 title += " "
             title += "(corrected)"
 
-        plt = main(dataDir, visit, title, args.outputTxtFile,
-                   gridPoints=args.gridPoints,
-                   showFwhm=args.showFwhm, minFwhm=args.minFwhm, maxFwhm=args.maxFwhm,
-                   correctDistortion=args.correctDistortion,
-                   showEllipticity=args.showEllipticity, ellipticityDirection=args.ellipticityDirection,
-                   showNdataFwhm=args.showNdataFwhm, showNdataEll=args.showNdataEll,
-                   minNdata=args.minNdata, maxNdata=args.maxNdata,
-                   verbose=args.verbose)
+        plot = main(dataDir, visit, title, args.outputTxtFile,
```

Comment:
This one was causing trouble because `plt` was clashing with the `plt` that was imported from `matplotlib`.